client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -336,27 +340,25 @@
 
 	.plan-features-2023-grid__pricing {
```

Comment:
WDYT about consolidating all the `plan-features-2023-grid__pricing` styles into one location while at it. So either move everything from here into `header-price` (which most look like overrides of `PlanPrice`) or move the [new styles](https://github.com/Automattic/wp-calypso/pull/74961/files#diff-0b8c173504dad73d4da3beb1c69d587a74c61893c99961bc0544b5bd506eb1b7R15-R22) here?